### DISCUSSION TITLE
[Idea/Feature Request] Support overriding right and left screen edge swipe (want Quick Actions on right swipe like Win10)

### DISCUSSION THREAD

**Discussion by madmrmax** [valinet/ExplorerPatcher#2409] [Ideas]
Today with and without EP, we are stuck with Windows 11 not being that friendly for tablet users to easily get to the Quick Actions UX with a swipe. In Windows 10 we could easily swipe from the right side to get to things like screen brightness. In Windows 11 swiping from bottom right corner is not touch friendly. 

I would love to see a few overrides available for right or left screen swipe.

**Comment by XiaoTDM** [comment]
Nevertheless, after the 24H2 update, you can swipe up from the bottom of the screen to open 'Start.' At this point, the taskbar will also be visible, allowing you to switch, close apps, or open 'Task View.'
